kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/client/DiscoveryClientProvider.scala

```diff
@@ -42,4 +43,19 @@ object DiscoveryClientProvider extends Logging {
     }
   }
 
+  def createDiscoveryClient(conf: KyuubiConf): DiscoveryClient = {
+    val classLoader = Thread.currentThread.getContextClassLoader
+    val className = conf.get(HighAvailabilityConf.DISCOVERY_CLIENT_CLASS)
+    if (className.isEmpty) {
```

Comment:
no need to check again as a result of https://github.com/apache/incubator-kyuubi/pull/2601/files#diff-884de93f6e9662de661d5ac820b28123d405bdf473912fc015cb1f84b70e8550R167

Reply:
This check still valid in case people manually set the configure value to empty string?

Reply:
I guess it is never reached

Reply:
Maybe, "className.isEmpty" is a simple check, should not hurt to keep it, in case other parts of the code have issue and do not deal with that config value properly.

Reply:
ok